DPGAnalysis/Skims/python/WElectronSkim_cff.py

```diff
@@ -104,10 +104,22 @@
     hltTags = cms.untracked.string( HLTPath ),
     triggerEventTag = cms.untracked.InputTag("hltTriggerSummaryAOD","",HLTProcessName),
     triggerResultsTag = cms.untracked.InputTag("TriggerResults","",HLTProcessName),
-    stageL1Trigger = cms.uint32(1)
+    stageL1Trigger = cms.uint32(1),
+    # Stage-1 L1T inputs
+    l1GtRecordInputTag = cms.InputTag('gtDigis'),
+    l1GtReadoutRecordInputTag = cms.InputTag('gtDigis'),
 )
 from Configuration.Eras.Modifier_stage2L1Trigger_cff import stage2L1Trigger
-stage2L1Trigger.toModify(PassingHLT, stageL1Trigger = 2)
+#stage2L1Trigger.toModify(PassingHLT, stageL1Trigger = 2)
+stage2L1Trigger.toModify(PassingHLT,
+    stageL1Trigger = 2,
+    # Stage-2 L1T inputs
+    l1tAlgBlkInputTag = cms.InputTag('gtStage2Digis'),
```

Comment:
l1tAlgBlkInputTag = 'gtStage2Digis',

Reply:
Isn't ```l1tAlgBlkInputTag``` an InputTag?

Reply:
Are they not defined before? I just look that it passes the existing config and modify. If that is the case, you don't need to redefine the type.

Reply:
Yes, they are defined in [here](https://github.com/cms-sw/cmssw/blob/master/L1Trigger/L1TGlobal/src/L1TGlobalUtilHelper.cc#L10L11). The ```l1tAlgBlkInputTag``` and ```l1tExtBlkInputTag``` will pass through WElectronSKim -> HLTPrescaleProvider -> L1TGlobalUtil-> L1TGlobalUtilHelper.

Reply:
> Are they not defined before?

No [*]. Please keep the patch as suggested.

[*] This is the starting module. It does not have `l1tAlgBlkInputTag` nor `l1tExtBlkInputTag`.
```
PassingHLT = cms.EDProducer("trgMatchGsfElectronProducer",    
    InputProducer = cms.InputTag( ELECTRON_COLL ),                          
    hltTags = cms.untracked.string( HLTPath ),
    triggerEventTag = cms.untracked.InputTag("hltTriggerSummaryAOD","",HLTProcessName),
    triggerResultsTag = cms.untracked.InputTag("TriggerResults","",HLTProcessName),
    stageL1Trigger = cms.uint32(1),
    # Stage-1 L1T inputs
    l1GtRecordInputTag = cms.InputTag('gtDigis'),
    l1GtReadoutRecordInputTag = cms.InputTag('gtDigis'),
)
```